src/web/components/Input/RadioInput.tsx

```diff
@@ -34,9 +34,14 @@ export function RadioInput<
         aria-invalid={error ? 'true' : 'false'}
         onValueChange={field.onChange}
       >
-        {options.map(({ optionLabel, value }) => (
+        {options.map(({ optionLabel, value, disabled }) => (
```

Comment:
I think we should update the storybook to include options that are disabled. Also, it seems like we are missing some styling for the disabled option (e.g. greyed out and maybe `cursor: not-allowed`

Reply:
I added styling and a test for disabled.